assets/css/main.css

```
@@ -329,14 +332,15 @@
 @layer components {
-  /* Gradient primary button */
+  /* Solid primary button */
   .btn-primary {
     @apply inline-flex items-center justify-center gap-2;
-    @apply font-normal text-sm text-black;
+    @apply font-normal text-sm;
     @apply px-5 py-2.5 rounded-xl;
     @apply transition-all duration-300 ease-out;
     @apply disabled:opacity-50 disabled:cursor-not-allowed disabled:transform-none disabled:shadow-none;
-    background: linear-gradient(135deg, var(--accent) 0%, var(--accent-dark) 100%);
+    background: var(--accent);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)